portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,98 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+
+			DMDocument.enableActionsMenuOnPortlet();
+
+			LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			DMDocument.tearDownCP();
+
+			DMFolder.tearDownCP();
+
+			PagesAdmin.tearDownCP();
+		}
+	}
+
+	@description = "Check that is possible to delete a Document and Media"
+	@priority = 4
+	test CanDeleteDM {
+		var dmEntryTitle1 = "DMTest 1";
+		var dmEntryFile1 = "Document_1.txt";
+		var dmEntryTitle2 = "DMTest 2";
+		var dmEntryFile2 = "Document_2.doc";
+
+		task ("And create 2 Documents and Media (Upload the file)") {
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = ${dmEntryTitle1},
+				groupName = "Guest",
+				mimeType = "text/plain",
+				sourceFileName = ${dmEntryFile1});
+
+			JSONDocument.addFileWithUploadedFile(
+				dmDocumentTitle = ${dmEntryTitle2},
+				groupName = "Guest",
+				mimeType = "application/msword",
+				sourceFileName = ${dmEntryFile2});
```

Comment:
Hey, you could also put this inside a for loop, since you are repeating code (what you did is not wrong, don't worry :slightly_smiling_face:).
Create a for loop to iterate inside a list of the elements ex: DMTest ${var}, your var could be the numbers 1 and 2. 
If you have any questions, don't hesitate to ask for help.

Reply:
Okay, changed it. Thanks for the tip!